registry.json

```diff
@@ -668,5 +668,20 @@
  "telegram": "rabbitholepools",                                            
  "operatorName": "Adrem",                                
  "github": "adrem1"                                               
- } 
-] 
+ },
+ {
+ "ticker": "AFRIK",                                                      
+ "poolId": "pool1l5t2lytxxx7jn0ujd5ssmfzaf24g2gjzjeddqmqjgun0z7r3gwv", 
```

Comment:
Your pool id is : fd16af916631bd29bf926d210da45d4aaa852242965ad06c124726f1 

I know confusing because on the ledger it is called pool Hash....

Reply:
Already changed by @leo42 thanks!